packages/core/src/application/mod.rs

```diff
@@ -0,0 +1,13 @@
+pub mod add_capability;
+pub mod delete_kittynode;
+pub mod delete_package;
+pub mod get_capabilities;
+pub mod get_installed_packages;
+pub mod get_packages;
+pub mod get_server_url;
+pub mod get_system_info;
+pub mod init_kittynode;
+pub mod install_package;
+pub mod is_docker_running;
+pub mod remove_capability;
+pub mod set_server_url;
```

Comment:
I don't think having all use cases / vertical slices under `application/*` will scale. Perhaps we should have a folder per domain object? Should all use cases always operate under a domains? If true, then a folder per domain could make sense here.